content/meetups.json

```
@@ -436,6 +436,14 @@
     "country": "DE",
     "state": ["Rheinland-Pfalz"]
   },
+  {
+    "name": "Einundzwanzig Karlsruhe",
+    "url": "https://t.me/+QmHNuOTtz_lmOTli",
+    "top": 63,
+    "left": 25,
+    "country": "DE",
+    "state": ["Baden-Württemberg"]
+  },
```

```suggestion
  {
    "name": "Bitcoin Karlsruhe",
    "url": "https://www.meetup.com/BitcoinKarlsruhe/",
    "top": 63,
    "left": 25,
    "country": "DE",
    "state": ["Baden-Württemberg"]
  },
  {
    "name": "Einundzwanzig Karlsruhe",
    "url": "https://t.me/+QmHNuOTtz_lmOTli",
    "top": 65,
    "left": 25,
    "country": "DE",
    "state": ["Baden-Württemberg"]
  },
```

siehe https://github.com/Einundzwanzig-Podcast/einundzwanzig.space/blob/661ada85a15651098e13e2ec1cb740d7c99fcde1/content/meetups.json#L43-L58

Vielleicht sollte man die Orange/Grau Icons durch Telegram/Meetup icons ersetzen